docs/contribution-guide.md

```diff
@@ -432,13 +432,29 @@ appropriate in a post-processing step.
 
 ## Submitting a pull request
 
-Once you've got the code and checked that the tests all pass as shown above,
-create a branch based on `master` with a descriptive name such as
-`fix-binary-operator-statements` or `add-support-for-soaked-member-access`.
+If you have found a bug or have a feature you'd like to add, it is probably
+worth creating an issue to document the bug or discuss the feature before doing
+significant work. Once you've decided on an approach, get the code and check
+that the tests all pass as shown above, create a branch based on `master` with
+a descriptive name such as `fix-binary-operator-statements` or `add-support-for-soaked-member-access`.
 Make changes to fix your bug or add your feature, ensuring that you write tests
 covering the changes. The tests you add or change *should fail* on `master`.
 
-Once you're satisfied with your changes create a pull request on your own fork.
+### Commit messages
+
+We use [semantic-release] to do releases, which means we follow the [Angular commit message guidelines](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit). You might want to get a passing familiarity with those, but the gist is this:
```

Comment:
I think you meant to make "[semantic-release]" a link? Doesn't seem to be a link when I do the markdown preview. Also, might be good to phrase it in a way that makes it clear that every "fix" or "feat" commit automatically generates a release, which was a totally new concept for me when I first started doing decaffeinate stuff.